pom.xml

```diff
@@ -170,6 +170,26 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>co.cask</groupId>
+          <artifactId>cdap-maven-plugin</artifactId>
+          <version>1.0.0-SNAPSHOT</version>
```

Comment:
i think we should be consistent with what version we use here. I think in most projects we use 1.1.0 or something